src/components/newsBox/newsBox.module.css

```diff
@@ -39,6 +43,12 @@
 }
 
 .desc {
-  color: rgb(137, 136, 136);
+  opacity: 0.6;
+  font-weight: lighter;
+}
+
+.createdAt {
+  color: rgba(137, 136, 136, 0.662);
```

Comment:
will it work for both dark/light themes?

Reply:
implemented changes